app/machinedriver_data.go

```diff
@@ -42,22 +48,31 @@ func addMachineDrivers(management *config.ManagementContext) error {
 		"c31b9da2c977e70c2eeee5279123a95d", []string{"ecs.aliyuncs.com"}, false, false, management); err != nil {
 		return err
 	}
-
-	return addMachineDriver("vmwarevsphere", "local://", "", nil, true, true, management)
+	return nil
 }
 
 func addMachineDriver(name, url, checksum string, whitelist []string, active, builtin bool, management *config.ManagementContext) error {
 	lister := management.Management.NodeDrivers("").Controller().Lister()
 	cli := management.Management.NodeDrivers("")
 	m, _ := lister.Get("", name)
+	annotations := map[string]string{}
```

Comment:
can we elaborate in the pr description on how annotations are used (to identify the credential filed, etc etc)